### PR TITLE
Publish Helm charts

### PR DESCRIPTION
**Pull request by niqdev**
No Travis integration in this PR

### CODE REVIEW

File: argocd/Chart.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 apiVersion: v1
 description: Argo CD Helm chart for Kubernetes
 name: argocd
-version: 1.0.0
+# <OFFICIAL_SEMVER_VERSION>-<CHART_VERSION>
+version: 1.0.0-0
```
Comment (conorfennell) [inline argocd/Chart.yaml:5]: So the left semver version matches the version of the application and the chart version is always one number?

File: README.md [hunk 2]
```diff
@@ -1 +1,17 @@
 # helm-charts
+
+```bash
+# add helm repository
+helm repo add el-public https://edgelevel.github.io/helm-charts
```
Comment (conorfennell) [inline README.md]: what is `el-public`?

File: dev.txt [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# init gh-pages
```
Comment (conorfennell) [inline dev.txt:1]: Whats `dex.txt`?

File: argocd/Chart.yaml (hunk 1 above)
Comment (niqdev) [reply argocd/Chart.yaml:5]: yep, `OFFICIAL_SEMVER_VERSION` should match the [official](https://github.com/argoproj/argo-cd/releases) version, while `<CHART_VERSION>` is just a sequence number that we should increase every time we make a change to our chart and we want to release it. I don't see the needs to have an internal semver. Does it make sense?

Comment (conorfennell) [reply argocd/Chart.yaml:5]: 👍

**Review by conorfennell — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (niqdev) [reply README.md]: It's the name that I propose to give to this repo, at the moment we have 2 helm repositories
* [helm-charts](https://edgelevel.github.io/helm-charts/) (this repo) which I propose to call `el-public`
* [gitops-k8s](https://edgelevel.github.io/gitops-k8s) (usage [example](https://github.com/niqdev/do-k8s/blob/master/bootstrap/apply.sh#L14)) which I propose to call `el-gitops`
The naming, as far as I know, is completely ignored and not used directly, in the requirements at least

`el-public` cos tomorrow we could potentially have `el-private`. Also, `edgelevel` or `helm-charts` are too generic and could cause issues if already added locally for another repo

File: dev.txt (hunk 3 above)
Comment (niqdev) [reply dev.txt:1]: Internal for development purposes only, to keep track of things that needs to be done, ideas, links or useful things for future developments that cannot be added to the readme or to an issue. 

It's a general approach that I use in several of my repos

**Comment by niqdev** [comment]
@conorfennell [here](https://github.com/niqdev/do-k8s/blob/master/.travis.yml) you can see an example of how I integrated Travis.
We need to create a gmail account, github user and travis account to make it properly.

File: README.md (hunk 2 above)
Comment (conorfennell) [reply README.md]: No need to abbreviate, just use `edgelevel-public`, I was confused by `el`.

You can go with whatever you want.

File: dev.txt (hunk 3 above)
Comment (conorfennell) [reply dev.txt:1]: It seems like clutter i.e. Nic personal notes. should be structured into README.md or removed.

Comment (conorfennell) [reply dev.txt:1]: But also PR is approved, it is just confusing for someone else besides yourself.

File: README.md (hunk 2 above)
Comment (niqdev) [reply README.md]: changed

**Pull request merged**